src/components/Identicon/StatusIcon.tsx

```diff
@@ -20,7 +25,37 @@ const IconWrapper = styled.div<{ size?: number }>`
   `};
 `
 
+const SockContainer = styled.div`
+  position: absolute;
+  background-color: ${colors.pink400};
+  display: flex;
+  justify-content: center;
+  border-radius: 50%;
+  width: 16px;
+  height: 16px;
+  bottom: -5px;
+  right: -5px;
+`
+
+const SockImg = styled.img`
+  width: 7.5px;
+  height: 10px;
+  margin-top: 3px;
+`
+
+const Socks = () => {
+  return (
+    <SockContainer>
+      <SockImg color="black" src={sockImg} />
```

Comment:
Why not use the svg directly, by importing it as a react component?

Reply:
I had style the svg, so I used a styled component